ledger/participant-integration-api/src/main/scala/platform/indexer/ha/HaCoordinator.scala

```diff
@@ -182,6 +190,11 @@ object HaCoordinator {
       }
     }
   }
+
+  class LockBusyException(lockId: LockId, lockMode: LockMode) extends RuntimeException {
+    override def getMessage: String =
+      s"Cannot acquire lock $lockId in lock-mode $lockMode: lock busy"
```

Comment:
What does it mean for a lock to be busy? Does it have to do with busy waiting?

```suggestion
class CouldNotAcquireDbLockException(lockId: LockId, lockMode: LockMode) extends RuntimeException {
    override def getMessage: String =  
        s"Could not acquire lock $lockId in lock-mode $lockMode"
```